less/components/modal.less

```diff
@@ -88,6 +91,77 @@
         background: saturate(@orangeSidebarColor, 10%);
       }
     }
+    .btn.btn-awsm {
```

Comment:
I don't really like this entire chunk of code as it's redundant and it overcomplicates our CSS button rulesets.  However, I think we should keep the mixed button styling issue separated from this PR since it involves decision from the designer as well as changes to many other existing buttons.  I have created https://github.com/mozilla/teach.webmaker.org/issues/1045 for this matter.